Modules/BookingManager/BookingProcess/class.ilBookingAssignParticipantsTableGUI.php

```diff
@@ -79,12 +79,10 @@ public function __construct(
 
 
         if ($this->pool->getScheduleType() == ilObjBookingPool::TYPE_NO_SCHEDULE) {
-            ilUtil::sendInfo(
-                sprintf(
-                    $this->lng->txt("book_objects_available"),
-                    ilBookingReservation::numAvailableFromObjectNoSchedule($a_booking_obj_id)
-                )
-            );
+            $GLOBALS['DIC']->ui()->mainTemplate()->setOnScreenMessage('info', sprintf(
```

Comment:
Can we avoid those $GLOBALS? There is a $DIC is already available in this constructor.